minidump-common/src/format.rs

```diff
@@ -842,6 +843,27 @@ pub enum ArmRegisterNumbers {
     ProgramCounter = 15,
 }
 
+/// aarch64 floating point state (old)
+#[derive(Clone, Copy, Pread, SizeWith)]
+pub struct FLOATING_SAVE_AREA_ARM64_OLD {
```

Comment:
Oh, sorry, I was looking at a single changeset and didn't see this in the next commit...